src/jaxsim/mujoco/visualizer.py

```diff
@@ -35,30 +37,120 @@ def __init__(
             **kwargs: Additional arguments for the renderer.
         """
 
-        width = width if width is not None else model.vis.global_.offwidth
-        height = height if height is not None else model.vis.global_.offheight
+        if isinstance(model, mj.MjModel):
+            single_model = model
+        elif isinstance(model, list) and len(model) == 1:
+            single_model = model[0]
+        else:
+            raise ValueError(
+                "Model must be a single instance of mj.MjModel or a list with at least one element."
+            )
+
+        width = width if width is not None else single_model.vis.global_.offwidth
+        height = height if height is not None else single_model.vis.global_.offheight
 
-        if model.vis.global_.offwidth != width:
-            model.vis.global_.offwidth = width
+        if single_model.vis.global_.offwidth != width:
+            single_model.vis.global_.offwidth = width
 
-        if model.vis.global_.offheight != height:
-            model.vis.global_.offheight = height
+        if single_model.vis.global_.offheight != height:
+            single_model.vis.global_.offheight = height
 
         self.fps = fps
         self.frames: list[npt.NDArray] = []
-        self.data: list[mujoco.MjData] | mujoco.MjData | None = None
-        self.model: mujoco.MjModel | None = None
+        self.data: list[mj.MjData] | mj.MjData | None = None
+        self.model: list[mj.MjModel] | mj.MjModel | None = None
```

Comment:
I think this PR also allows for visualizing multiple models. Can you update the description accordingly?